CHANGELOG.md

```diff
@@ -26,7 +26,7 @@
 * SECURITY
   * Update github.com/containerd/containerd (#978) (#980)
 * BUGFIXES
-  * Return to page after clikcing login at navbar (#975) (#976)
+  * Return to page after clicking login at navbar (#975) (#976)
```

Comment:
Some projects don't like changing changelogs. Happy to drop this or anything else